colorize.js

```diff
@@ -9,7 +9,11 @@
 
 /* global AnsiUp, localStorage */
 
-const colors = ['#FFD54F', '#BCAAA4', '#90CAF9', '#FFCC80', '#FBE9E7', '#B0BEC5', '#DCE775', '#A5D6A7', '#81C784', '#9FA8DA', '#E0E0E0', '#CFD8DC', '#F48FB1', '#F8BBD0', '#F0F4C3', '#C0CA33', '#CE93D8', '#80CBC4', '#E1BEE7', '#CDDC39', '#C5CAE9', '#EF9A9A', '#FFFF00', '#B2EBF2', '#BDBDBD', '#FFE57F', '#B2DFDB', '#BBDEFB', '#69F0AE', '#FFCDD2', '#9CCC65', '#80DEEA', '#76FF03', '#B2FF59', '#C8E6C9']
+const colors = ['#FFD54F', '#BCAAA4', '#90CAF9', '#FFCC80', '#FBE9E7', '#B0BEC5', '#DCE775', '#A5D6A7', '#81C784', '#9FA8DA', '#E0E0E0', '#CFD8DC', '#F48FB1', '#F8BBD0', '#F0F4C3', '#C0CA33', '#CE93D8', '#80CBC4', '#E1BEE7', '#CDDC39', '#C5CAE9', '#EF9A9A', '#FFFF00', '#B2EBF2', '#BDBDBD', '#FFE57F', '#B2DFDB', '#BBDEFB', '#69F0AE', '#FFCDD2', '#9CCC65', '#80DEEA', '#76FF03', '#B2FF59', '#C8E6C9'];
+const errorLevel = "level\":\"error";
```

Comment:
I'm not sure that, this format covers many cases, in terms of if there is whitespace between level and error.
I think we can look a couple of log samples to extract more general format.
What do you think?

Reply:
agreed, so to keep it generic as in other places, looking for keywords as 'info' or 'error' now in a case-insensitive way and apply color.

Reply:
![image](https://user-images.githubusercontent.com/10493402/90342077-2243c800-dfba-11ea-9d1e-5ea1644f92d7.png)

Reply:
output is looking as above now

Reply:
Why there is two different color for level:INFO? Second one loaded later?

Reply:
yes, when we scroll, it picks up a new color

Reply:
OK, maybe we should apply fixed color value for each level?

Reply:
do you mean apply a fixed color to diff levels like we are doing for "error" right now 
e.g.   let color = '#FF4500' (line 333) in PR 
or there is a way to retain what color which was picked for 'info' before scroll happened and use the same.

Reply:
Yes, I implied your first suggestion. 
There is also a way to retain same color but even if we keep the same color during scroll, we cannot accomplish it in the case of page refresh.

We can pick soft, bright color info level, or maybe we can ignore info for colorize. what do you think?

Reply:
I guess assigning some light shade to info levels as well looks better otherwise it looks empty or incomplete I should say :) 
Please check out these snapshots ... 
light blue -> info, 
light yellow -> warning 
light red -> error 

![Screen Shot 2020-08-24 at 9 12 17 PM](https://user-images.githubusercontent.com/10493402/91122364-03df6b80-e64f-11ea-99f4-b5b33d8c20f4.png)
![Screen Shot 2020-08-24 at 9 11 47 PM](https://user-images.githubusercontent.com/10493402/91122377-0b067980-e64f-11ea-8f5c-882233d41db6.png)